kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -408,7 +416,7 @@ extension [A, S, E](effect: A < (Abort[Absent] & S))
       *   A computation that produces the result of this computation with the Abort[Absent] effect translated to Choice
       */
     def emptyAbortToChoice(using Flat[A], Frame): A < (S & Choice) =
```

Comment:
Can the evidence be in the extension? that should clean up signatures throughout... or does it reduce clarity of why a method is unavailable?

Reply:
1. IDEs cannot resolve the methods as quickly
2. The error messages are generally harder to understand when it's on the extension methods